src/components/Author/Author.test.tsx

```diff
@@ -1,30 +1,21 @@
 import React from 'react';
 import { mount } from 'enzyme';
-import Authors from './Author';
 
-const mockPackageMeta = jest.fn(() => ({
-  latest: {
-    homepage: 'https://verdaccio.tld',
-    bugs: {
-      url: 'https://verdaccio.tld/bugs',
-    },
-    dist: {
-      tarball: 'https://verdaccio.tld/download',
-    },
-  },
-}));
+import { DetailContext } from '../../pages/Version';
 
-jest.mock('../../pages/Version', () => ({
-  DetailContextConsumer: component => {
-    return component.children({ packageMeta: mockPackageMeta() });
-  },
-}));
+import Authors from './Author';
 
 describe('<Author /> component', () => {
   beforeEach(() => {
     jest.resetAllMocks();
   });
 
+  const component = (packageMeta: React.ContextType<typeof DetailContext>['packageMeta']): JSX.Element => (
```

Comment:
Hmmm, as it is a component, it must be capitalize and have a more descriptive name

Reply:
I would also write this component outside of the describe<'Author .....

Reply:
`component` ==> `mockAuthorComponent` . It has to be lower case since is a function. Other popular naming way might be `withAuthorComponent`.

Reply:
Hmm unfortunately I don't agree with you @juanpicado it returns an JSX element and according to the standards it should be capitalize...Please see that also Function Components are functions

Reply:
I think the main difference is that this doesn't take props, you can't use it as a JSX element.

That's why I didn't consider it a component and why I used lowercase, it's more of a function that returns JSX.

Reply:
@priscilawebdev any last comment here? or can we merge?

Reply:
@juanpicado no...We can merge :-)